registry.json

```diff
@@ -5096,5 +5096,18 @@
    "telegram": "wafflepool",
    "operatorName": "",
    "github": ""
+ },
+ {
+   "ticker": "TOKEN",
+   "poolId": "1e5f4eebc56a6726fd36af2f870b3bd62b204f3cd1b36e32e8a56737",
+   "social": {
+     "twitter": "token_stakepool",
+     "facebook": "",  
+     "instagram": "",
+     "youtube": ""
+   },
+   "telegram": "tokenstakepool",
```

Comment:
Please change this to the telegram username of an individual and not a group or channel

Reply:
Hi @Bez625 

I managed to delete the tokenstakepool channel and open an account with a new SIM card. 
It has the same name: tokenstakepool
Thank you.